ui/pages/settings/security-tab/delete-metametrics-data-button/delete-metametrics-data-button.test.tsx

```diff
@@ -81,6 +85,9 @@ describe('DeleteMetaMetricsDataButton', () => {
       if (selector === getMetaMetricsDataDeletionStatus) {
         return 'INITIALIZED';
       }
+      if (selector === getParticipateInMetaMetrics) {
```

Comment:
Added due to https://github.com/MetaMask/metamask-extension/pull/29081/files#r1913396674